tests/client.spec.ts

```diff
@@ -1,7 +1,8 @@
-import {client} from '../src/helpers/client';
-import {asperaDesktop} from '../src/index';
 import {mockFetch} from './mocks';
+import {asperaDesktop} from '../src';
+import Client from '../src/helpers/client';
 
+const client = new Client();
```

Comment:
If we still export a singleton you can revert these changes.

Reply:
Done!